sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TablesImpl.java

```diff
@@ -68,10 +68,10 @@
      */
     @Host("{url}")
     @ServiceInterface(name = "AzureTableTables")
-    private interface TablesService {
+    public interface TablesService {
         @Get("/Tables")
         @ExpectedResponses({200})
-        @UnexpectedResponseExceptionType(TableServiceErrorException.class)
+        @UnexpectedResponseExceptionType(HttpResponseException.class)
```

Comment:
Did the Swagger change the error type here? Or was this error type changed manually in the past?

Reply:
This is odd because all of the other methods still use  `TableServiceErrorException`. Maybe it's an issue in the swagger?

Reply:
I rolled back this change for the time being and will investigate for next month's release.